clhep-toolfile.spec

```diff
@@ -30,6 +30,7 @@ cat << \EOF_TOOLFILE >%i/etc/scram.d/clhepheader.xml
   <runtime name="CLHEP_PARAM_PATH" value="$CLHEP_BASE"/>
   <runtime name="CMSSW_FWLITE_INCLUDE_PATH" value="$INCLUDE" type="path"/>
   <runtime name="ROOT_INCLUDE_PATH"  value="$INCLUDE" type="path"/>
+  <runtime name="CLING_MODULEMAP_PATH" value="$PCM_UTIL_BASE/include" type="path"/>
```

Comment:
why PCM_UTIL_BASE ?

Reply:
@davidlange6 , can you please fix this? You need to use $CLHEP_BASE instead of $PCM_UTIL_BASE :-)

Reply:
whoops